src/amo/css/ScreenShots.scss

```diff
@@ -7,8 +7,14 @@ $screenshot-height: 220px;
 $image-height: $screenshot-height - 20px;
 $screenshot-width: 320px;
 
-.ScreenShots .pswp-thumbnails {
-  line-height: 0;
+.ScreenShots {
+  .pswp-thumbnails {
+    line-height: 0;
```

Comment:
Why is it needed?

Reply:
do you mean the `line-height` style?  This was on the line above I just moved it so I could nest the caption style as well

Reply:
heh. 🤕